Misc/NEWS.d/next/Library/2019-01-14-11-53-10.bpo-34294.3JFdg2.rst

```diff
@@ -0,0 +1 @@
+re module, fix wrong capturing groups in rare cases. Failed SRE(match) doesn't reset capturing groups, we need to reset them between two SRE(match) callings in loops.
```

Comment:
`SRE(match)` is an implementation detail. Please reword a news entry in terms of user visible functions.